bodhi/server/config.py

```diff
@@ -563,21 +586,25 @@ class BodhiConfig(dict):
     }
 
     def __getitem__(self, *args, **kw):
+        """Ensure the config is loaded, and then call the superclass __getiten__."""
```

Comment:
getitem?

Reply:
@stickster I've fixed the typo in a new commit I've pushed - look good to you?